src/test/java/org/exmple/utility/UtilityMethodsTest.java

```diff
@@ -96,4 +100,12 @@ public void countEven_shouldThrowNullPointerException() {
 
         assertTrue(actualMessage.contains(expectedMessage));
     }
+
+    @Test
+    public void reverse() {
```

Comment:
For naming unit tests we use the pattern `[method name that is being tested]_[expected result]`. For your test case it may look like: `reverse_shouldReverseArray`